src/content/docs/billing/manage-plans/add-manage-plan-groups.mdx

```diff
@@ -0,0 +1,51 @@
+---
+page_id: adee7343-3d87-4045-9d62-5db57fce6596
+title: Add and manage plan groups
+sidebar:
+  order: 5
+relatedArticles:
+  - 88e1773a-b681-441f-b4c7-d7d339116867
+  - fe9fea0c-274c-4d6b-9cc5-eccdbaad85b7
+  - 37d5ae8e-cfd1-4e5d-8333-387b6967ec23
+---
+
+Plan groups are a way of organizing sets of related plans together. 
+
+A plan group can be defined as an 'organization' or 'user' type, depending who you are selling to. For (B2B) businesses, use the organization type. For (B2C) individual customers, use the 'user' type. If you are a B2B2C business, create one group of each type. 
+
+How you set up plans in the group impacts how you present feature bundles on a pricing table.
+
+## Default plan groups
+
+When you first add a plan in Kinde, you select the plan type and a plan group will be automatically created for you. You can later change the name of the group, or make a different group the default group. There can only be one default group for organizations and one for users. The default group is what is displayed by default on a pricing table.
+
+## Add a plan group
+
+1. Go to **Billing > Plans**.
+2. Select **Add group**.
+3. Enter a **Name** and select the type of plan the group contains.
+4. Choose a colour. This only applies to how the group appears on the **Plans** page.
+5. Select **Save**.
+6. Repeat for all the groups you want to add. 
+
+## Organize plans within a group
+
+The order of plans within a group is important, as it represent the order plans will appear on the pricing table. You don't have to set them up in order when you are first adding them, but you will need to move them into order before showing users the pricing table. 
```

Comment:
@clairekinde11 this isn't quite correct. The pricing table actually is where the order within the pricing table is set.

The only place at the moment that this order comes into play is for inside Kinde admin if you are manually assigning an org/user to a plan, it's the order displayed there, as there is no pricing table.